Lib/test/test_bytes.py

```diff
@@ -1382,6 +1382,19 @@ def test_bytearray_api(self):
             except OSError:
                 pass
 
+    def test_mod_use_after_free(self):
+        # Prevent UAF in bytearray % (a1, a2) with re-entrant a[12].__repr__.
+        # Regression test for https://github.com/python/cpython/issues/142557.
+        fmt = bytearray(b"%a %a")
+
+        class S:
+            def __repr__(self):
+                fmt.clear()
+                return "E"
+
+        args = (S(), S())
```

Comment:
It can perhaps be reproduced even with a single argument (if there is something after its format unit).

Reply:
Yes, a single argument is actually enough.